content/en/docs/instrumentation/net/getting-started/metrics.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Metrics
+description: Getting Started with OpenTelemetry .NET Metrics
+weight: 4
+---
+
```

Comment:
This should have a description about OTel metrics with .NET and how it's based on a core library, similar to tracing